http-server/src/test/java/org/triplea/server/lobby/game/listing/GameListingEventQueueTest.java

```diff
@@ -140,7 +124,7 @@ void gameUpdated() {
                 GameListingMessageType.GAME_UPDATED.name(), lobbyGameListing)));
     assertThat(
         "Verify closed session is removed",
-        gameListingEventQueue.getSessions().values(),
+        gameListingEventQueue.getSessionSet().values(),
         hasSize(1));
   }
 }
```

Comment:
Slightly out-of-scope, but could you re-do the unchecked warning suppression from https://github.com/triplea-game/triplea/pull/5802/files#diff-748e01ab941a190e4392897c626bf28aR72-R116 ?

Reply:
Bump